Sources/StreamVideoSwiftUI/CallingViews/iOS13/BackportStateObject.swift

```diff
@@ -41,10 +41,12 @@ public struct BackportStateObject<ObjectType: ObservableObject & Sendable>: @pre
         self.thunk = thunk
     }
 
-    public mutating func update() {
-        // Not sure what this does but we'll just forward it
-        _state.update()
-        _observedObject.update()
+    nonisolated public func update() {
```

Comment:
probably good to test this on iOS 13? 😭

Reply:
I don't have a way to test this. Xcode doesn't allow me to download any version below iOS 15 and i don't own a device on that version.

Reply:
So, by disabling the @available and forcing the use of the PrejoiningView_iOS13 i'm able to run the app and join a call on both Xcode 15 and 16 when compiling the app for iOS 15.5 (the lowest i can do)